decidim-proposals/app/controllers/decidim/proposals/admin/proposals_controller.rb

```diff
@@ -124,13 +124,26 @@ def update_scope
         def edit
           enforce_permission_to :edit, :proposal, proposal: proposal
           @form = form(Admin::ProposalForm).from_model(proposal)
-          @form.attachment = form(AttachmentForm).from_params({})
+          document = proposal.documents.first
+          @form.attachment = if document.present?
+                               form(AttachmentForm).from_params({ file: document.file, title: translated_attribute(document.title) })
+                             else
+                               form(AttachmentForm).from_params({})
+                             end
```

Comment:
This seems like it should belong to `Admin::ProposalForm#map_model`, can you move it there, please?